src/containers/UserSettings/settings.ts

```diff
@@ -75,6 +76,16 @@ export const languageSetting: SettingProps = {
     },
 };
 
+export const binaryDataInPlainTextDisplay: SettingProps = {
+    settingKey: BINARY_DATA_IN_PLAIN_TEXT_DISPLAY,
+    title: i18n('settings.binaryDataInPlainTextDisplay.title'),
+    description:
```

Comment:
My idea is to display description only in monitoring mode because in embedded mode this feature will only be provided with newer YDB versions.

Perhaps it would be better to move this condition somewhere, where is it better?

Reply:
I think this condition is ok, I'd move it to some helper function `isMultiClusterVersion()`

Reply:
Moved to src/containers/ClusterModeGuard/ClusterModeGuard.tsx.

Reply:
I think, name `AppModeGuard` will suit more here

Reply:
I think base64 displaying should be the same throughout the application, not just in query. So I would make/name this setting very common.

Reply:
It's not clear to me what "app mode" means. I would assume that this is about prod/test envs.

Reply:
You're right